### ISSUE TITLE
Duplicates

### ISSUE THREAD

**Issue by AdamDanischewski** [mklement0/whichpm#1]
I tried this out after seeing you post about it on StackOverflow. It works but throws a warning about duplicates and lists whatever I search for 7+ times. I haven't looked at your code to see what's going on but it looks like it's on the whichpm side.  

Here's an example on MooX::Aliases: 
```
whichpm: WARNING: DUPLICATE module files found for 'MooX::Aliases':
  /home/adanisch/perl5/lib/perl5/MooX/Aliases.pm
  /home/adanisch/perl5/lib/perl5/MooX/Aliases.pm
  /home/adanisch/perl5/lib/perl5/MooX/Aliases.pm
  /home/adanisch/perl5/lib/perl5/MooX/Aliases.pm
  /home/adanisch/perl5/lib/perl5/MooX/Aliases.pm
  /home/adanisch/perl5/lib/perl5/MooX/Aliases.pm
  /home/adanisch/perl5/lib/perl5/MooX/Aliases.pm
  /home/adanisch/perl5/lib/perl5/MooX/Aliases.pm
/home/adanisch/perl5/lib/perl5/MooX/Aliases.pm
```
That's what it does for everything I search. I'm running the binary whichpm on Ubuntu Linux. 
whichpm version:  v0.1.7

**Comment by mklement0** [comment]
Intriguing - I see the problem neither on macOS 10.15.1 nor on Ubuntu 18.04.

The warning definitely comes from `whichpm` (which is a Perl script), and you get it by default if duplicates are found - unless you add `-a` to explicitly ask for duplicates.

What Ubuntu version with what Perl version do you see this on?

**Comment by AdamDanischewski** [comment]
If I add the  `-a` option I get the same thing except without the warning on the first line. The perl version is "v5.28.1 built for x86_64-linux-gnu-thread-multi".   
Distributor ID: Ubuntu
Description:    Ubuntu 19.10
Release:        19.10
Codename:       eoan

**Comment by mklement0** [comment]
I suspect that this is not related to the Ubuntu version after all, and that you simply have duplicate entries in your effective `@INC` value (module search path).

I've just published v0.2 to the npm registry, which now doesn't list duplicate modules stemming from such duplicate search-path entries anymore, and simply _warns_ - as before, you can suppress the warning with option `-q`.

I'm assuming this resolves the issue; let me know if it doesn't.
